web-registry/graphql.schema.json

```diff
@@ -1579,6 +1579,98 @@
         ],
         "possibleTypes": null
       },
+      {
+        "kind": "INPUT_OBJECT",
+        "name": "AddAddrToAccountInput",
```

Comment:
i'm a bit confused why this is here. NBD though, i just wanted to add a note.

Reply:
Hmm good question. There seems to be a lot in here auto-generated that has nothing to do with the simple onChainId add. This is produced by the `yarn graphql:codegen` command when there is an api change.

Reply:
it must just be that it hasn't got into v4 branch yet. i'm pretty sure that these changes are in the master branch though. i guess since you guys are merging master into v4, this might disappear in this diff soon (if i understand correctly)

Reply:
@wgwz `yarn graph:codegen` targets your local server… so these are being generated from whichever local environment @mhagel is running